service/templates/notifications.html

```diff
@@ -55,6 +55,16 @@
                 </md-radio-group>
               </td>
             </tr>
+            <tr>
+              <td>Spawnhost Expiration</td>
+              <td colspan="3">
+                <md-radio-group layout="row" style="width:100%" ng-model="settings.notifications.spawn_host_expiration" md-no-ink="true">
+                  <md-radio-button value="email"></md-radio-button>
+                  <md-radio-button value="slack" ng-disabled='!settings.slack_username || settings.slack_username == ""'></md-radio-button>
+                  <md-radio-button value=""></md-radio-button>
```

Comment:
To preserve compatibility with the existing functionality, can we change things so that email is default?